views/includes/head.ejs

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title><%= pageTitle %></title>
+    <link rel="stylesheet" href="/css/main.css">
```

Comment:
where is `</head>`?

Reply:
In shop.ejs and 404.ejs and admin.ejs file , there is </head> . we include the head.ejs file there by using include() . Now its have compelete ejs code . 

Refer this link : https://blog.soshace.com/en/programming-en/express-js-ejs-templating/